launcher/ui/themes/CatPack.cpp

```diff
@@ -117,5 +120,21 @@ QString JsonCatPack::path(QDate now)
         if (startDate <= now && now <= endDate)
             return var.path;
     }
-    return m_defaultPath;
+    auto dInfo = QFileInfo(m_default_path);
+    if (!dInfo.isDir())
+        return m_default_path;
+
+    QStringList supportedImageFormats;
+    for (auto format : QImageReader::supportedImageFormats()) {
+        supportedImageFormats.append("*." + format);
+    }
+
+    auto files = QDir(m_default_path).entryInfoList(supportedImageFormats, QDir::Files, QDir::Name);
+    if (files.length() == 0)
+        return "";
+    auto idx = now.dayOfYear() % files.length();
```

Comment:
```suggestion
    auto idx = (now.dayOfYear() - 1) % files.length();
```
First day = 1, first file = 0, that doesn't make sense and the you will always see the cat of tommorow